app/src/sharedTest/java/org/oppia/android/app/databinding/DrawableBindingAdaptersTest.kt

```diff
@@ -0,0 +1,235 @@
+    return it.findViewById(R.id.view_for_drawable_binding_adapters_test)
+  }
+
+  private inline fun <reified V, A : Activity> ActivityScenario<A>.runWithActivity(
+    crossinline action: (A) -> V
+  ): V {
+    // Use Mockito to ensure the routine is actually executed before returning the result.
+    @Suppress("UNCHECKED_CAST") // The unsafe cast is necessary to make the routine generic.
+    val fakeMock: Consumer<V> = mock(Consumer::class.java) as Consumer<V>
+    val valueCaptor = ArgumentCaptor.forClass(V::class.java)
+    onActivity { fakeMock.consume(action(it)) }
+    verify(fakeMock).consume(valueCaptor.capture())
+    return valueCaptor.value
+  }
+
+  private fun setUpTestApplicationComponent() {
+    ApplicationProvider.getApplicationContext<DrawableBindingAdaptersTest.TestApplication>()
+      .inject(this)
+  }
+
+  // TODO(#59): Figure out a way to reuse modules instead of needing to re-declare them.
+  @Singleton
+  @Component(
+    modules = [
+      RobolectricModule::class,
+      PlatformParameterModule::class, PlatformParameterSingletonModule::class,
+      TestDispatcherModule::class, ApplicationModule::class,
+      LoggerModule::class, ContinueModule::class, FractionInputModule::class,
+      ItemSelectionInputModule::class, MultipleChoiceInputModule::class,
+      NumberWithUnitsRuleModule::class, NumericInputRuleModule::class, TextInputRuleModule::class,
+      DragDropSortInputModule::class, ImageClickInputModule::class, InteractionsModule::class,
+      GcsResourceModule::class, TestImageLoaderModule::class, ImageParsingModule::class,
+      HtmlParserEntityTypeModule::class, QuestionModule::class, TestLogReportingModule::class,
+      AccessibilityTestModule::class, LogStorageModule::class, CachingTestModule::class,
+      PrimeTopicAssetsControllerModule::class, ExpirationMetaDataRetrieverModule::class,
+      ViewBindingShimModule::class, RatioInputModule::class, WorkManagerConfigurationModule::class,
+      ApplicationStartupListenerModule::class, LogUploadWorkerModule::class,
+      HintsAndSolutionConfigModule::class, HintsAndSolutionProdModule::class,
+      FirebaseLogUploaderModule::class, FakeOppiaClockModule::class, PracticeTabModule::class,
+      DeveloperOptionsStarterModule::class, DeveloperOptionsModule::class,
+      ExplorationStorageModule::class, NetworkModule::class, NetworkConfigProdModule::class,
+      NetworkConnectionUtilDebugModule::class, NetworkConnectionDebugUtilModule::class,
+      AssetModule::class, LocaleProdModule::class, ActivityRecreatorTestModule::class
+    ]
+  )
+  /** Create a TestApplicationComponent. */
+  interface TestApplicationComponent : ApplicationComponent {
+    /** Build the TestApplicationComponent. */
+    @Component.Builder
+    interface Builder : ApplicationComponent.Builder
+
+    /** Inject [DrawableBindingAdaptersTest] in TestApplicationComponent . */
+    fun inject(drawableBindingAdaptersTest: DrawableBindingAdaptersTest)
+  }
+
+  /**
```

Comment:
Please remove all KDocs from this test except the top-level one.

They're optional, and I think the current ones don't add a whole lot (since they're mostly explaining what should already be understood given that this code is boilerplate).

Reply:
Yes! I have removed unnecessary KDocs.